internal/services/network/client/client.go

```diff
@@ -143,8 +145,14 @@ func NewClient(o *common.ClientOptions) (*Client, error) {
 	ServiceEndpointPolicyDefinitionsClient := network.NewServiceEndpointPolicyDefinitionsClientWithBaseURI(o.ResourceManagerEndpoint, o.SubscriptionId)
 	o.ConfigureClient(&ServiceEndpointPolicyDefinitionsClient.Client, o.ResourceManagerAuthorizer)
 
-	SubnetsClient := network.NewSubnetsClientWithBaseURI(o.ResourceManagerEndpoint, o.SubscriptionId)
-	o.ConfigureClient(&SubnetsClient.Client, o.ResourceManagerAuthorizer)
+	LegacySubnetsClient := network.NewSubnetsClientWithBaseURI(o.ResourceManagerEndpoint, o.SubscriptionId)
```

Comment:
Given the Virtual Resources that exist using different API versions will be problematic during updates - can we remove `LegacySubnetsClient` here and update _all_ usages to use the new `SubnetsClient`?

However, given we're importing the Meta Client for API Version `2023-06-01` - it'd be better to instead update _that_ to the latest API Version - and then update usages of the Subnets client - this ensures the issue described above won't happen - would you be able to take a look into fixing that?

Reply:
OK. Got it. I intend to close this PR and submit another PR for it.